ls1intum/Artemis
# `Quiz exercises`: Fix batch quiz editing when batch is added

### Checklist
#### General


#### Client

### Motivation and Context



[#5958](https://github.com/ls1intum/Artemis/issues/5958)

https://user-images.githubusercontent.com/10885503/207876795-1db3c723-75a2-4d6c-9a52-3d09faa6cbe5.mov

### Description


- Disabled edit button if batch is added for batched quiz

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Course

1. Log in to Artemis
2. Navigate to Course Administration
3. Select the Course
4. Click Exercises
5. Create Quiz Exercise
6. Set to Batch
7. Click Save
8. Click Add Batch
9. Verify that Edit button is disabled

### Review Progress






#### Code Review
#### Manual Tests

### Test Coverage



| Class/File | Branch | Line |
|------------|-------:|-----:|
| quiz-exercise.component.ts | 63.41% | 88.61% |

### Screenshots



https://user-images.githubusercontent.com/10885503/207878483-ff3a6b94-8bec-45d5-8fc4-64920e020173.mov
